### PR TITLE
Mongo fix

### PR DESCRIPTION
**Pull request by gcollin**
## Problem

- Mongo only supports CPUs with avx

## Solution

- Install older version if cpu is incorrect
## PR Status

- [x] Code finished and ready to be reviewed/tested
- [x] The fix/enhancement were manually tested (if applicable)

## Automatic tests

Automatic tests can be triggered on https://ci-apps-dev.yunohost.org/ *after creating the PR*, by commenting "!testme", "!gogogadgetoci" or "By the power of systemd, I invoke The Great App CI to test this Pull Request!". (N.B. : for this to work you need to be a member of the Yunohost-Apps organization)

### CODE REVIEW

**Comment by gcollin** [comment]
!testme

**Comment by gcollin** [comment]
!testme

**Comment by gcollin** [comment]
!testme

**Comment by gcollin** [comment]
!testme

**Comment by gcollin** [comment]
!testme

File: scripts/ynh_mongo_db__2 [hunk 1]
```diff
@@ -313,18 +313,23 @@ ynh_install_mongo() {
 	mongo_version="${mongo_version:-$YNH_MONGO_VERSION}"
 
 	ynh_print_info --message="Installing MongoDB Community Edition ..."
-	ynh_install_extra_app_dependencies --repo="deb http://repo.mongodb.org/apt/debian $(ynh_get_debian_release)/mongodb-org/$mongo_version main" --package="mongodb-org mongodb-org-server mongodb-org-tools mongodb-mongosh" --key="https://www.mongodb.org/static/pgp/server-$mongo_version.asc"
+	# Install the version that works with the host cpu (see https://docs.mongodb.com/manual/administration/production-notes/#x86_64)
+	local mongo_debian_release=$(ynh_get_debian_release)
+
+	if [[ $(cat /proc/cpuinfo) != *"avx"* && "$mongo_version" != "4.4" ]]; then
+    ynh_print_error --message="The version of Mongo you're trying to install may not be compatible with your cpu (Missing avx instruction set)."
```
Comment (ericgaspar) [inline scripts/ynh_mongo_db__2:320]:
```suggestion
    ynh_print_err --message="The version of Mongo you're trying to install may not be compatible with your cpu (Missing avx instruction set)."
```
https://yunohost.org/oc/packaging_apps_helpers#ynh-print-err

**Comment by gcollin** [comment]
!testme

**Comment by gcollin** [comment]
!testme

**Pull request merged**